examples/apps/generic_client/eth_generic_client.py

```diff
@@ -78,8 +76,7 @@ def _parse_command_line(args):
         "-b", "--blockchain",
         help="Blockchain type to use in proxy model",
         choices={"fabric", "ethereum"},
```

Comment:
Can fabric be removed from choices?

Reply:
Deliberately left there @manju956 .  Pretty soon all the generic client would be merged. Then this would make sense. The other pieces of code removed in this PR were stale.

Reply:
Thanks for the clarity.